### ISSUE TITLE
Null type mismatch in AudioManagerImpl.java

### ISSUE THREAD

**Issue by GiovannaTheo** [eclipse/smarthome#5242]
The method:

```java 
@Override
public void stream(String url, String sinkId) throws AudioException {
    AudioStream audioStream = url != null ? new URLAudioStream(url) : null;
    play(audioStream, sinkId);
}
```

shows "Null type mismatch: required '@NonNull AudioStream' but the provided value 
 is inferred as @Nullable" in Eclipse while setting-up the SmartHome IDE following Eclipse documentation (https://www.eclipse.org/smarthome/documentation/development/ide.html). 

This result in an error that prevents running on Eclipse, unless problems severities are adjusted.

I believe the method could be replaced by:

```java    
@Override
public void stream(String url, String sinkId) throws AudioException {
    Objects.requireNonNull(url, "URL should not be null.");
     AudioStream audioStream = new URLAudioStream(url);
     play(audioStream, sinkId);
}
```

Any thought on this?

**Comment by kaikreuzer** [comment]
I'd say this has just been fixed this minute: https://github.com/eclipse/smarthome/pull/5241

**Comment by htreu** [comment]
Hey @GiovannaTheo, good to see you are already getting familiar with the IDE and the code!
In case the fix from #5241 fixes it for you please close this issue.
